exasol_transformers_extension/deployment/scripts_deployer_cli.py

```diff
@@ -46,11 +47,13 @@ def scripts_deployer_main(
         ssl_client_private_key: str,
         use_ssl_cert_validation: bool,
         schema: str,
-        language_alias: str):
+        language_alias: str,
+        use_spans: bool): #todo do we even want to offer it via the cli?
```

Comment:
This a real good question, at max only for us internally for the moment. We might remove it at the end of the PR